src/databricks/labs/ucx/account/workspaces.py

```diff
@@ -76,9 +76,11 @@ def sync_workspace_info(self, workspaces: list[Workspace] | None = None):
             except (PermissionDenied, NotFound, ValueError):
                 logger.warning(f"Failed to save workspace info for {ws.config.host}")
 
-    def create_account_level_groups(self, prompts: Prompts, workspace_ids: list[int] | None = None):
+    def create_account_level_groups(self, prompts: Prompts):
         acc_groups = self._get_account_groups()
-        workspace_ids = self._get_valid_workspaces_ids(workspace_ids)
+        workspace_ids = [workspace.workspace_id for workspace in self._workspaces()]
+        if not workspace_ids:
+            raise ValueError("No workspace ids provided in the configuration found in the account")
```

Comment:
I see the message has not changed, however, I think it is inaccurate. There is not such thing as a configuration in the **account**. Could you update it to be accurate?